soh/soh/Enhancements/randomizer/randomizer.h

```diff
@@ -38,9 +43,11 @@ class Randomizer {
     std::string GetAdultAltarText() const;
     std::string GetGanonText() const;
     std::string GetGanonHintText() const;
-    std::string GetHintFromCheck(RandomizerCheck check);
+    //CustomMessageEntry GetHintFromCheck(RandomizerCheck check);
     GetItemID GetRandomizedItemIdFromKnownCheck(RandomizerCheck randomizerCheck, GetItemID ogId);
     GetItemID GetRandomizedItemId(GetItemID ogId, s16 actorId, s16 actorParams, s16 sceneNum);
+    //std::string GetCustomGetItemMessage(GlobalContext* globalCtx, GetItemID giid);
```

Comment:
delete not comment out

Reply:
Done, plus a few other unused functions/definitions that were unused removed as well.